### ISSUE TITLE
Conditional lights / tempating to allow difference between night and day

### ISSUE THREAD

**Issue by Patrick1610** [fredck/lightener#17]
Hi, 

First of all I want to say that I really like this integration, because it makes is so much easier to easily operate the lights in one room. 
I started to play with the integration this week, and I was wondering what would be the best solution for the following. 
Some rooms have different settings during daytime and nighttime in my setup. For example, during the normal scene in my home, one light is 100% during the day, and 50% during the night. Another light never turns on during the day and is 100% during the night. 

I have more scenes that I can fit into the setup of the integration by assigning a percentage of the Lighener entity to each scene I currently have, but difference between day and night does not seem to be possible. 
Is it possible to do this with templating? ( I am not very good at tempating, so I lack the knowledge to know the possibilities)
Or is there another easy way to do this? The easiest way seems to be to make two Lighteners per room, and show the correct one on the lovelace dashboard by using templates to determine day/night, but I would like to have this in one lighener if possible. 

With kind regards, 
Patrick

**Comment by fredck** [comment]
At first, I thought that this is a feature that _could_ make sense, although unlikely to be introduced in the foreseeable future. But then I figured that this requirement can be applied to basically every integration out there.

To make it clearer, let me rephrase the request: The ability to change an (any) integration configuration based on a (any) condition.

I would hardly see this landing in Home Assistant itself, since this can be handled with implementors "creativity". For instance, the double Lightener idea you have is a good way to go with it. Then it is up to automations to decide which one to use, depending on any condition you want (sun, time, rain, presence, mood, etc.)

I'm closing this one here for now but feel invited to reopen it if you or anyone else have a different opinion so we can talk further about it.

Thanks!

**Comment by dotgrid** [comment]
@Patrick1610 I was thinking about this type of setup, too, since I'd like to achieve what you described. 

I can see what @fredck is saying. For sure. I wanted to clarify a detail, from my understanding of Fred's advice.

- We should configure both a **Day Lightener** and **Night Lightener**
- We create a _virtual light_ entity (this is the light that is added to Lovelace)
- We create an automation, triggered by changes to the _virtual light_ brightness
- The automation has conditional actions: during daytime, apply the _virtual light's_ brightness to our **Day Lightener**; at nighttime, apply the brightness to our **Night Lightener**

I hope I've understood Fred's advise correctly. And I hope this is useful to @Patrick1610 as well!

Thank you to the developers for creating what is basically my DREAM light control system! 🙂

**Comment by Patrick1610** [comment]
Hi @dotgrid 

I am using mostly automations that set my lighting completely automatic. Using the suggested approach with two entities per light, I would not create a virtual light myself. 
In the automations that automatically regulate my lighting I will make a condition based on sun down/up. 
On Lovelace I would show the day or night entity based on a conditional card. That way I can directly adjust the correct Lightener without another automation in between. 

Regards, Patrick
